SFS_config.m

```diff
@@ -421,6 +421,8 @@
 %                   see test_interpolation_methods.m in the validation folder. For
 %                   typical HRIRs with leading and trailing zeros, the error is
 %                   negligible.
+%   'timedomain'  - Interpolation in the time domain with cross-correlation for
+%                   estimation of TOA differences.
```

Comment:
For new users it might help if you write `time of arrival (TOA)` here instead of `TOA`

Reply:
Changed that as well, thanks for the comments!